### DISCUSSION TITLE
Sykz's submission (Prologue)

### DISCUSSION THREAD

**Discussion by Sykzpixel** [stellar/sorobanathon#99] [General]
https://litemint.com/items/GCVDYHT53A7J6GAQHEA5QEJTRWQXAWLRRCDAZYF7AVIOGEFZNBHIWOIA/Sykz3

**Comment by stellarjohn** [comment]
Hi Sykz!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
